uma_vasp/__init__.py

```diff
@@ -1,6 +1,7 @@
+from datetime import datetime, timezone
 from flask import Flask, request
 from lightspark import LightsparkSyncClient
-from uma import InMemoryPublicKeyCache
+from uma import InMemoryPublicKeyCache, InMemoryNonceCache, create_pubkey_response
```

Comment:
I think you might need to rebuild and upload the compiled sdk tar to get these errors fixed.